src/mmw/apps/modeling/tasks.py

```diff
@@ -178,9 +211,12 @@ def add_huc12(srat_huc12, aggregate):
         return {
             'Loads': loads,
             'SummaryLoads': summary_loads,
-            'Catchments': {comid: format_catchment(result)
-                           for comid, result
-                           in srat_huc12['catchments'].items()},
+            'Catchments': {
+                comid: format_catchment(result, catchment_loads_template)
+                for comid, result
+                in srat_huc12['catchments'].items()
+                },
```

Comment:
This looks neater and still satisfies the linter:

```suggestion
            'Catchments': {
                comid: format_catchment(result, catchment_loads_template)
                for comid, result in srat_huc12['catchments'].items()},
```